api_examples/insert_institution_csv.py

```diff
@@ -160,12 +160,21 @@ def get_build_args(self, argv):
             action="store_true",
             default=False,
         )
+        parser.add_argument(
+            "--append-tags",
+            required=False,
+            help="This tags will be appended to each imported institution. "
+            "You can use | (pipe) character to append several tags.",
```

Comment:
Why `pipe`? Web-ui use `,` for that and CSV library handles escaping.

Reply:
I know, we already talked about this. I decided to use this format for my convenience. When I work with plain csv file then this way it's just more readable for me.

And by the way, I need to rewrite most of this script, because it's originated from another script and ended looking quite chaotic.